src/components/BookingForm/SubscriptionPackagesStep/SubscriptionPackages.jsx

```diff
@@ -48,65 +68,88 @@ const SubscriptionPackages = () => {
   };
 
   return (
-    <Box>
-      <SubscriptionPkgsContainer>
-        {packages.map((pkg, index) => (
-          <SubscriptionPackagesCard
-            image={bg}
-            color={colors[index]}
-            packageType={pkg.name}
-            descriptionItems={[
-              { label: "Price", value: pkg.price, highlighted: true },
-              { label: "Duration", value: pkg.duration, highlighted: false },
-            ]}
-            selected={form.formData.selectedPackage?.id === pkg.id}
-            onClick={() => handleClick(pkg, pkg)}
-            key={index}
-          />
-        ))}
+      <SubscriptionPkgsContainer isMobile={isMobile}>
+        {isMobile ? (
+            <Swiper
+                spaceBetween={0.5}
+                slidesPerView={2.2}
+                style={{ width: "100%" }}
+            >
+              {packages.map((pkg, index) => (
+                  <SwiperSlide key={index}>
+                    <SubscriptionPackagesCard
+                        image={bg}
+                        color={colors[index]}
+                        packageType={pkg.name}
+                        descriptionItems={[
+                          { label: "Price", value: pkg.price, highlighted: true },
+                          { label: "Duration", value: pkg.duration, highlighted: false },
+                        ]}
+                        selected={form.formData.selectedPackage?.id === pkg.id}
+                        onClick={() => handleClick(pkg, pkg)}
+                        key={index}
+                    />
+                  </SwiperSlide>
+              ))}
+            </Swiper>
+        ) : (
+            packages.map((pkg, index) => (
+                <SubscriptionPackagesCard
+                    key={index}
+                    image={bg}
+                    color={colors[index]}
+                    packageType={pkg.name}
+                    descriptionItems={[
+                      { label: "Price", value: pkg.price, highlighted: true },
+                      { label: "Duration", value: pkg.duration, highlighted: false },
+                    ]}
+                    selected={form.formData.selectedPackage?.id === pkg.id}
+                    onClick={() => handleClick(pkg, pkg)}
+                />
+            ))
+        )}
       </SubscriptionPkgsContainer>
-    </Box>
   );
 };
 
 export default SubscriptionPackages;
 
 const SubscriptionPackagesCard = ({
-  image,
-  color,
-  packageType,
-  descriptionItems,
-  onClick,
-  selected = false,
-}) => {
+                                    image,
+                                    color,
+                                    packageType,
+                                    descriptionItems,
+                                    onClick,
+                                    selected = false,
+                                  }) => {
   return (
```

Comment:
Please use a good formatter to avoid these kinds of changes, making it harder for someone reviewing to check blame and creating unnecessary lines to reveiw